Client/Frontend/Settings/ThemeSettings/ThemeSettingsController.swift

```diff
@@ -79,6 +79,7 @@ class ThemeSettingsController: ThemedTableViewController, StoreSubscriber {
     func newState(state: ThemeSettingsState) {
         themeState = state
         tableView.reloadData()
+        applyTheme()
```

Comment:
Add the call to applyTheme because the ThemeManager could have changed the current theme in response to user brightness changed or system brightness changed